### ISSUE TITLE
Unhandled reference to navigator in node environment

### ISSUE THREAD

**Issue by rdumont** [vtex/splunkevents-js#8]
```
C:\Users\rdumont\projects\courier\node_modules\splunk-events\dist\splunk-events.js:153
                userAgent: navigator.userAgent,
                           ^

ReferenceError: navigator is not defined
    at SplunkEvents.getAdditionalInfo (C:\Users\rdumont\projects\courier\node_modules\splunk-events\dist\splunk-events.js:153:21)
    at SplunkEvents.logEvent (C:\Users\rdumont\projects\courier\node_modules\splunk-events\dist\splunk-events.js:119:50)
    at logEvent (logger.js:50:16)
    at Object.info (logger.js:70:47)
    at Server.<anonymous> (index.js:36:10)
    at Server.g (events.js:286:16)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:185:7)
    at emitListeningNT (net.js:1279:10)
    at _combinedTickCallback (internal/process/next_tick.js:71:11)
```

**Comment by nevescode** [comment]
@rdumont this was fixed in this branch https://github.com/vtex/splunkevents-js/tree/feature/node-support

I'm waiting for the fix of #5 to merge.

**Comment by nevescode** [comment]
Fixed in #11
